### ISSUE TITLE
caption role should be associated with aria-labelledby not describedby

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1085]
In lieu of future work to finalize the `caption` role, one thing I would like to get squared away for ARIA 1.2 is the current definition stating:
>Authors SHOULD reference the element with role caption by setting `aria-describedby` on the figure or table.

We should change this for the 1.2 release as `caption` should provide the `table` or `figure` its accessible name, not a description.

edit: ooops.... clearly was in the wrong repo when i made the issue...
